js/map.js

```diff
@@ -49,28 +49,59 @@ const geoButtonsSelector = '#geography-select button'
 const catButtonsSelector = '#category-select button'
 const statButtonsSelector = '#status-select button'
 
-function getTooltip(props){
+class InfoControl {
+  constructor() {
+    this.container = document.createElement('div')
+    this.container.className = 'info'
+    this.container.innerHTML = 'Hover over a feature to see details'
+  }
+
+  onAdd(map) {
+    this.map = map
+    return this.container
+  }
+
+  onRemove() {
+    this.container.parentNode.removeChild(this.container)
+    this.map = undefined
+  }
+
+  updateInfo(content) {
```

Comment:
I wish this was Typescript, but since it's not, typed doc blocks might be helpful (e.g. `/** @param content {string} The CSS display style to use */`)